plugin/evm/gossiper.go

```diff
@@ -43,30 +45,31 @@ const (
 
 // Gossiper handles outgoing gossip of transactions
 type Gossiper interface {
-	// GossipTxs sends AppGossip message containing the given [txs]
-	GossipTxs(txs []*types.Transaction) error
+	// GossipEthTxs sends AppGossip message containing the given [txs]
```

Comment:
In Coreth we had AtomisTxs/EthTxs, when we migrated it to Subnet-EVM we (I) had renamed it to Txs since there is no AtomicTxs. But after some recent syncs I was thinking it is probably better if we just kept them same. You're correct this could have been done in some other PR.